source/manual/fall-back-to-mirror.html.md

```diff
@@ -63,20 +63,36 @@ cd /mnt/crawler_worker/www.gov.uk
 
 ## Forcing failover to the static mirrors
 
-Because the CDN will retry every request against the mirrors automatically if origin is unavailable,
-[stopping Nginx on the cache machines with Fabric][fab-fail] will result in falling back to mirrors:
+Because the CDN will retry every request against the mirrors automatically if origin is unavailable, stopping Nginx on the cache machines will result in falling back to mirrors:
+
+You will need to SSH into each box in turn and run the following commands.
+
+SSH into the first with:
 
-```bash
-$ fab $environment class:cache incident.fail_to_mirror
+```
+gds govuk connect -e production ssh cache:1
 ```
 
-To disable the fallback:
+and run:
 
-```bash
-$ fab $environment class:cache incident.recover_origin
+```
+  sudo govuk_puppet --test --disable "fail_to_mirror task (by $USER)"
```

Comment:
There's no need to use `sudo` with `govuk_puppet`

Also it would be good to give the command to run to re-enable puppet as well